tox.ini

```diff
@@ -18,6 +25,16 @@ basepython =
     py27: python2.7
     py33: python3.3
     py34: python3.4
+commands =
+    {envpython} --version
+    - coverage erase
+    coverage run test_settings.py
+    - coverage report
+    gulp tests:unit
+    gulp tests:lint
+    - sh test_requirements/runserver.sh
+    sleep 60
+    gulp tests:integration
 
 [testenv:pep8]
```

Comment:
I agree, flake8 covers everything PEP8 does, but also does quite a lot of deeper static analysis.